SAKPaaS/src/app/components/locate-button/locate-button.component.scss

```diff
@@ -24,10 +24,6 @@ button {
 @media (max-width: 600px) {
   button {
     right: 7.5vw;
-  }
-}
-@media (max-height: 800px) {
-  button {
-    top: 240px;
+    bottom: calc(7vh + 20px);
   }
```

Comment:
```suggestion
  }
  .map[_ngcontent-vvb-c82] {
      padding-bottom: 50px;
      height: calc(30vh + 50px);
  }
  .ol-unselectable, .ol-viewport {
      overflow: visible !important;
   }
```

Reply:
`30vh` because thats the `top` of the location-panel on mobile.

Reply:
Makes the map Overlap by exactly 50px with the location-panel and hinders the overlays from doing so.